src/s3fs_cred.cpp

```diff
@@ -113,21 +113,19 @@ bool UpdateS3fsCredential(char** ppaccess_key_id, char** ppserect_access_key, ch
 //-------------------------------------------------------------------
 // Class Variables
 //-------------------------------------------------------------------
-const char* S3fsCred::ALLBUCKET_FIELDS_TYPE     = "";
-const char* S3fsCred::KEYVAL_FIELDS_TYPE        = "\t";
-const char* S3fsCred::AWS_ACCESSKEYID           = "AWSAccessKeyId";
-const char* S3fsCred::AWS_SECRETKEY             = "AWSSecretKey";
-
-const int   S3fsCred::IAM_EXPIRE_MERGIN         = 20 * 60;              // update timing
-const char* S3fsCred::ECS_IAM_ENV_VAR           = "AWS_CONTAINER_CREDENTIALS_RELATIVE_URI";
-const char* S3fsCred::IAMCRED_ACCESSKEYID       = "AccessKeyId";
-const char* S3fsCred::IAMCRED_SECRETACCESSKEY   = "SecretAccessKey";
-const char* S3fsCred::IAMCRED_ROLEARN           = "RoleArn";
-
-const char* S3fsCred::IAMv2_token_url           = "http://169.254.169.254/latest/api/token";
-int         S3fsCred::IAMv2_token_ttl           = 21600;
-const char* S3fsCred::IAMv2_token_ttl_hdr       = "X-aws-ec2-metadata-token-ttl-seconds";
-const char* S3fsCred::IAMv2_token_hdr           = "X-aws-ec2-metadata-token";
```

Comment:
Previously these were not truly `const` -- `IAMv2_token_hdr` could be reassigned to another `const char *`

Reply:
Please explain in more detail what you mean by this comment.

Reply:
`IAMv2_token_hdr = "foo"` later is legal.  `const char *` only means that the data pointed to must be constant, not the variable itself.  `const char * const`  or `const char[]` prevent this.

Reply:
Thanks